frontend/src/services/createPostService.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Might be better to just `export default submitPost` below the declaration instead of exporting it as non-default and suppressing the warning?

Reply:
True that. I was probably prematurely generalising it.

Reply:
We can easily change it back if we start exporting multiple things from here (I suspect once we start doing that the warning would go away anyway).